wagon-providers/wagon-http-shared/src/main/java/org/apache/maven/wagon/shared/http/AbstractHttpClientWagon.java

```diff
@@ -354,6 +360,92 @@ private static PoolingHttpClientConnectionManager createConnManager()
         return connManager;
     }
 
+    /**
+     * The type of the retry handler, default to DefaultHttpRequestRetryHandler.
+     * Values can be default, standard (StandardHttpRequestRetryHandler), or a fully qualified name class.
```

Comment:
This is inconsistent in style. Please use: `label` (`class name`), or a fully qualified class name.